### PR TITLE
Fix part of #2480: Fixes 2 Audio Tests

### PR DESCRIPTION
**Pull request by theMr17**
## Explanation

Fixes part of #2480
- Fixes `testAudioWifi_ratioExp_audioIcon_audioFragHasDefaultLangAndAutoPlays()` by checking default language at the end and testing audio icons in a better way.
- Fixes `testAudioWifi_ratioExp_continueInteraction_audioButton_submitAns_feedbackAudioPlays()` by scrolling to the _**continue_interaction_button**_ before performing click on it , as it was positioned at the lower end of the recycler view.

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-A11y-Guide))
- For PRs introducing new UI elements or color changes, both light and dark mode screenshots must be included
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing

### CODE REVIEW

**Comment by adhiamboperes** [comment]
@theMr17, could you please walk me through the solution that you are trying to achieve in this PR?

**Comment by theMr17** [comment]
@adhiamboperes, By this PR, I am trying to figure out exactly why the tests are failing in https://github.com/oppia/oppia-android/issues/2480 and finding a fix a for it.

File: app/src/sharedTest/java/org/oppia/android/app/player/exploration/ExplorationActivityTest.kt [hunk 1]
```diff
@@ -1181,15 +1178,10 @@ class ExplorationActivityTest {
 
   // TODO(#89): The ExplorationActivity takes time to finish. This test case is failing currently.
   @Test
-  @Ignore("The ExplorationActivity takes time to finish, needs to fixed in #89.")
+  // @Ignore("The ExplorationActivity takes time to finish, needs to fixed in #89.")
```
Comment (adhiamboperes) [inline app/src/sharedTest/java/org/oppia/android/app/player/exploration/ExplorationActivityTest.kt]: Do remove the comment

**Review by adhiamboperes — CHANGES_REQUESTED** [review]
@theMr17, this was not assigned to me, so I'm not sure it is ready for review, but I took a look anyway. 

The tests look good, with one minor comment.

Please also update the PR explanation with what you identified the problem as, and the fix you applied. Once you are ready for a full review, complete the PR checklist and convert too ready.

**Comment by theMr17** [comment]
@adhiamboperes PTAL

File: app/src/sharedTest/java/org/oppia/android/app/player/exploration/ExplorationActivityTest.kt [hunk 2]
```diff
@@ -1097,23 +1090,28 @@ class ExplorationActivityTest {
...
       onView(withId(R.id.play_pause_audio_icon)).check(
         matches(
-          withDrawable(
-            R.drawable.ic_pause_circle_filled_white_24dp
-          )
+          withDrawable(R.drawable.ic_pause_circle_filled_white_24dp)
         )
       )
+
+      onView(withText("What is a Ratio?"))
```
Comment (adhiamboperes) [inline app/src/sharedTest/java/org/oppia/android/app/player/exploration/ExplorationActivityTest.kt]: Whats the purpose of this? No assertion is being performed on the view. Same as in the test below.

**Review by adhiamboperes — CHANGES_REQUESTED** [review]

File: app/src/sharedTest/java/org/oppia/android/app/player/exploration/ExplorationActivityTest.kt (hunk 2 above)
Comment (theMr17) [reply app/src/sharedTest/java/org/oppia/android/app/player/exploration/ExplorationActivityTest.kt]: I have added assertion to it now. It checks for _"audioFragHasDefaultLang"_. And removed the line from the test below

**Review by adhiamboperes — APPROVED** [review]
Thanks @theMr17! This LGTM.

**Pull request merged**